### DISCUSSION TITLE
How do you separate credentials from the _settings.yml file so that you don't store them in version control?

### DISCUSSION THREAD

**Discussion by abertolli** [dadoonet/fscrawler#1914] [Q&A]
Is there a way to import an external file for credentials (in my case, elasticsearch preferably with the API key)?  We would like to have our fscrawler _settings.yml file in version control, but do not want that to contain credentials.  So far the only way I can tell to do this with the documentation is to hack around using --username and then use redirects to send the information on the commandline from another file.

**Comment by dadoonet** [comment]
I agree. I opened an issue to track it. I don't know yet how to do this though.
